.ci/travis/travis.sh

```diff
@@ -458,19 +458,48 @@ check-missing-pitests)
   ;;
 
 verify-no-exception-configs)
+  mkdir -p .ci-temp
   wget -q \
+    --directory-prefix .ci-temp \
+    --no-clobber \
     https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/checks-nonjavadoc-error.xml
   wget -q \
+    --directory-prefix .ci-temp \
+    --no-clobber \
     https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/checks-only-javadoc-error.xml
   MODULES_WITH_EXTERNAL_FILES="Filter|ImportControl"
   xmlstarlet sel --net --template -m .//module -v "@name" \
-    -n checks-nonjavadoc-error.xml -n checks-only-javadoc-error.xml \
+    -n .ci-temp/checks-nonjavadoc-error.xml -n .ci-temp/checks-only-javadoc-error.xml \
     | grep -vE $MODULES_WITH_EXTERNAL_FILES | grep -v "^$" \
-    | sort | uniq | sed "s/Check$//" > web.txt
+    | sort | uniq | sed "s/Check$//" > .ci-temp/web.txt
   xmlstarlet sel --net --template -m .//module -v "@name" -n config/checkstyle_checks.xml \
     | grep -vE $MODULES_WITH_EXTERNAL_FILES | grep -v "^$" \
-    | sort | uniq | sed "s/Check$//" > file.txt
-  diff -u web.txt file.txt
+    | sort | uniq | sed "s/Check$//" > .ci-temp/file.txt
+  DIFF_TEXT=$(diff -u .ci-temp/web.txt .ci-temp/file.txt | cat)
+  if [[ $DIFF_TEXT != "" ]]; then
+    if [[ $TRAVIS_PULL_REQUEST =~ ^([0-9]+)$ ]]; then
+      LINK_PR=https://api.github.com/repos/checkstyle/checkstyle/pulls/$TRAVIS_PULL_REQUEST
+      REGEXP="https://github.com/checkstyle/contribution/pull/"
```

Comment:
2 things:
1) It might be nice if there is a check that the PR in other repo actually fulfills missing Check(s).
2) When merged and master CI runs, it would be nice if we check that the referenced PR is already merged, otherwise master should fail.

Reply:
https://github.com/checkstyle/checkstyle/issues/7304

>  it would be nice if we check that the referenced PR is already merged

as this validation is executed on master too, it will fail. 
As it looks for PR only when there is a diff between files in master of both repositories.